src/components/HumburgerMenu/Humburger.css

```diff
@@ -0,0 +1,25 @@
+.humburger {
+    position: fixed;
+    left: 3vw;
+    top: 3vw;
+    width: 2rem;
+    height: 2rem;
+    padding: 0;
+    background: transparent;
```

Comment:
not sure that is the color that you want to see. 
if the menu is visible the background is not transparent

Reply:
I deleted. Done!